server/lib/configuration.js

```diff
@@ -48,6 +48,13 @@ var conf = module.exports = convict({
     default: path.resolve(__dirname, '..', '..', 'cert.pem'),
     doc: 'The location of the SSL certificate in pem format'
   },
+  client_metrics: {
+    enabled: {
```

Comment:
`client_metrics.enabled` implies a global on/off switch of client metrics, whereas this flag only enables/disables output to stderr. Maybe this should be something like:

```js
client_metrics: {
  stderr_collector_enabled: {
     ...
  },
  statsd_collector_enabled: {
    ...
  }
}
```

I could also see something like: 
```js
client_metrics: {
  collectors: {
     default: [ 'ga', 'stderr', 'statsd' ],
     doc: 'Where to send client metrics'
  }
}
```